conda/plugins/subcommands/doctor/health_checks.py

```diff
@@ -0,0 +1,88 @@
+# Copyright (C) 2012 Anaconda, Inc
+# SPDX-License-Identifier: BSD-3-Clause
+from __future__ import annotations
+
+import json
+from pathlib import Path
+
+OK_MARK = "✅"
+REPORT_TITLE = "\nENVIRONMENT HEALTH REPORT\n"
+DETAILED_REPORT_TITLE = "\nDETAILED ENVIRONMENT HEALTH REPORT\n"
+MISSING_FILES_SUCCESS_MESSAGE = f"{OK_MARK} There are no packages with missing files.\n"
+
+
+def display_report_heading(prefix: str) -> None:
+    """
+    Displays our report heading
+    """
+    environment = Path(prefix)
+    environment_name = environment.name
+    print("-" * 20)
+    print(REPORT_TITLE)
+    print(f"Environment Name: {environment_name}\n")
+
+
+def get_number_of_missing_files(prefix: str) -> dict[str, int]:
+    """
+    Returns a dictionary with packages and the number of missing files in them
+    """
+    packages_with_missing_files = find_packages_with_missing_files(prefix)
+
+    return {k: len(v) for k, v in packages_with_missing_files.items()}
```

Comment:
This helper function doesn't provide anything meaningful to us. Simply calling `find_packages_with_missing_files` and doing the `len` inline in `display_health_checks` below is sufficient and will be more readable.